modAionImpl/src/org/aion/zero/impl/blockchain/AionImpl.java

```diff
@@ -68,6 +67,7 @@ private AionImpl(boolean forTest) {
         blockchainCallbackInterfaces = Collections.synchronizedList(new ArrayList<>());
         aionHub.getPendingState().setPendingTxCallback(new PendingTxCallback(blockchainCallbackInterfaces));
         aionHub.getPendingState().setNetworkBestBlockNumberCallback(new NetworkBestBlockCallback(this));
+        aionHub.getPendingState().setTxBroadcastCallback(new TxBroadcastCallback(this));
```

Comment:
Similar to previous comments, I think this should be included in the pending state constructor.